pybossa/view/projects.py

```diff
@@ -1463,7 +1463,10 @@ def export_statuses(short_name, task_id):
         elif user_detail['lock_ttl']:
             user_detail['status'] = 'Locked'
 
-    tr_statuses = dict(redundancy=task.n_answers,
+    # gold tasks may have more answers than redundancy 1
+    # in rare case, task may have answers more than redundancy
```

Comment:
Looks like "1" at the end of 1466 is redundant :-) 
BTW, we won't have answers more than redundancy anymore, except for some existing data. I was thinking this can be fixed in pybossa-theme-default, but apparently, fixing in the server side is also a good option.

Reply:
No, "1" at line 1466 is  to mention > 1. Initially, I had similar though about themes, however, it would be better that server controls the data passed to the frontend, hence this fix. Thanks for the review!